its/ruling/src/test/resources/jdk6/squid-S3626.json

```diff
@@ -0,0 +1,73 @@
+{
+'jdk6:java/awt/AWTEventMulticaster.java':[
+922,
+],
+'jdk6:java/awt/geom/CubicCurve2D.java':[
+1286,
+],
+'jdk6:java/awt/geom/QuadCurve2D.java':[
+1027,
+],
+'jdk6:java/awt/image/PixelGrabber.java':[
+379,
+394,
+410,
+],
+'jdk6:java/beans/Introspector.java':[
+1333,
+],
+'jdk6:java/beans/beancontext/BeanContextServicesSupport.java':[
+621,
+636,
+],
+'jdk6:java/beans/beancontext/BeanContextSupport.java':[
+612,
+630,
+],
+'jdk6:java/lang/ref/Finalizer.java':[
+162,
+],
+'jdk6:java/lang/reflect/Proxy.java':[
+444,
+],
+'jdk6:java/math/BigDecimal.java':[
+3530,
+],
+'jdk6:java/math/MathContext.java':[
+130,
+152,
+],
+'jdk6:java/net/DatagramSocket.java':[
+699,
+],
+'jdk6:java/net/PlainSocketImpl.java':[
+172,
+201,
+518,
+],
+'jdk6:java/net/SocketPermission.java':[
+427,
+],
+'jdk6:java/security/AccessControlContext.java':[
+383,
+],
+'jdk6:java/security/IdentityScope.java':[
+67,
+],
+'jdk6:java/security/Policy.java':[
+309,
+],
+'jdk6:java/text/DecimalFormat.java':[
+2551,
```

Comment:
This two issues looks like a FPs to me, if you remove the `continue` statement, you are going to hit the `break` statement immediately after, which exit the loop.